src/Core/Request.php

```diff
@@ -85,13 +85,39 @@ public function createCallPayload(string $method, array $arguments) : \SimpleXML
         $call->addAttribute('Method', $method);
 
         //Add Arguments to the Call Node
-        foreach ($arguments as $key => $value) {
-            $call->addChild($key, $value);
-        }
+
+        $call = $this->sxmlAddChildrenRecursive($call,$arguments);
```

Comment:
You may want to wrap this with a try/catch since it can throw a YourMembershipException